provisioning/playbooks/roles/transactor_server/templates/DatastoreRpcService.conf.j2

```diff
@@ -0,0 +1,3 @@
+io.mediachain.datastore.rpc.port: 10002
```

Comment:
I think it's the right place to hardcoded, as we need to specify them somehow.
We could add a variable, but I don't think we need it rn.